code/game/turfs/simulated/mineral.dm

```diff
@@ -11,12 +11,8 @@
 /turf/simulated/floor/mineral/plasma
 	name = "plasma floor"
 	icon_state = "plasma"
-	floor_tile = /obj/item/stack/tile/mineral/plasma
```

Comment:
The runtime that happens when you do " returnToPool(/obj/item/stack/tile/mineral/plasma) "